keyboards/bastardkb/dilemma/3x5_2/keymaps/via/keymap.c

```diff
@@ -0,0 +1,229 @@
+/**
+ * Copyright 2022 Charly Delay <charly@codesink.dev> (@0xcharly)
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include QMK_KEYBOARD_H
+
+#ifdef DILEMMA_AUTO_POINTER_LAYER_TRIGGER_ENABLE
```

Comment:
It's only in develop, ATM, but end of month, the auto-mouse layer stuff will hit master. 

https://qmk.github.io/qmk_docs_devel/#/feature_pointing_device?id=pointing-device-auto-mouse

(eg, you don't need to do anything, but it may be worth thinking about/prepping for)

Reply:
Yup, excited to remove this custom code. I'm following the progress here and will make sure to use the core implementation when it gets merged in `master`.